app/blog/2018-03-20-polymer-3-latest-preview.md

```diff
@@ -243,7 +339,7 @@ import('my-view1.js').then((MyView1) => {
 
 Dynamic imports enable lazy-loading of resources, replacing the functionality that we previously used for this in Polymer 2.0 (`Polymer.importHref()`). 
 
-## Tools {#tools}
+## Tools updates {#tools}
 
 We have made changes to the Polymer CLI tools to support imports using package names. 
```

Comment:
In this section, I think we might want to remove the polymer build example (since it's not super useful).

And add a paragraph to the effect that the `polymer build` command has been updated to support names in module imports, but is still missing support for other build features, such as bundling modules and transforming modules for browsers that lack native module support.